### ISSUE TITLE
15 minutes = 0.2 hours?

### ISSUE THREAD

**Issue by dcd2** [projecthamster/hamster#332]
I just started using this handy little application, and so far I think it's the bee's knees; however, one thing stands out like a sore thumb: 15 minutes is considered 0.2 hours of work. Shouldn't that be 0.25 hours? Maybe there's some standard billing practise that I'm unaware concerning rounding?

Out of curiosity, I tried out a few combinations:
5 minutes (0.08333333 h) = 0.1 hours (rounds up)
10 minutes (0.16666667 h) = 0.2 hours (rounds up)
15 minutes (0.25 h) = 0.2 hours (rounds down)
20 minutes (0.33333333 h) = 0.2 hours (rounds down)
25 minutes (0.41666667 h) = 0.3 hours (rounds down)
30 minutes (0.5 h) = 0.5 hours (correct)

Basically, I'm losing time if I work in 15,20,25 minute increments, which is not cool as I like to work in 15 minute increments. Is there some way to specify how the rounding works, or is this a "feature"?

PS-this took me 10 minutes to write, so I will bill you 0.2 hours! JK! :)

PPS- Time Tracker 2.91.3 (Linux Mint 18/Ubuntu 16.04)

**Comment by elbenfreund** [comment]
Thank you for your feedback, it is much appreciated.

Project hamster and its various sub-components is currently undergoing some major changes. We prepare the introduction of a rewritten codebase for most of the underlying functionality. A direct consequence of this is that it is unlikely that any open/new bugs within the current/old codebase will be fixed (unless someone steps up and offers to do so) as most resources currently available will be invested in making the rewrite prime time ready.

If you are still interested in working on the new codebase (repositories: hamster-lib/cli/gtk/dbus) we would be most thrilled. Please feel free to either open a new issue with the relevant repository and/or join the discussion on the [mailinglist](http://lists.projecthamster.org/listinfo.cgi/hamster-dev-projecthamster.org).
For more context please see our [website](http://projecthamster.org).

Thanks for your interest and support! Eric.

**Comment by ederag** [comment]
Thanks for the report.
Current version works with hours/minutes, not decimal hours any longer, so this is fixed there.

**Comment by ederag** [comment]
Closing, since this issue is thought to be solved in the development version.
Thanks for contributing to hamster improvement !
